src/components/CreateVmWizard/steps/style.css

```diff
@@ -120,6 +120,12 @@
   margin: 0 3px;
 }
 
+.disk-bootable-edit {
+  display: flex;
+  margin: auto;
+  width: 50%;
+}
+
```

Comment:
Add another style for the checkbox to remove the ugly top/bottom margins and keep the row on a single line...

```css
.disk-bootable-edit :global(div.checkbox) {
  margin: 0;
  display: flex;
  align-items: center;
}
```

Note: the `:global()` bit is an instruction to the webpack CSS loader module to keep the rule selector as-is.  Otherwise the CSS loader generates unique names for things